### ISSUE TITLE
Integration doesn't work

### ISSUE THREAD

**Issue by csababalogh** [syssi/xiaomiplug#30]
Sorry, maybe a stupid question, but what did I wrong?
In the configuration.yaml this is the first Xiaomi Smart Plug (I have 3):

switch:
  - platform: xiaomi_miio_plug
    name: Xiaomi Plug (Night light)
    host: 10.0.1.34
    token: 4f1c00268527bdd5b49ea6145883f84c
    model: chuangmi.plug.hmi206

IP valid, token valid, model OK.

After restart in the Devices the only thing what I see is an entity.

![image](https://github.com/user-attachments/assets/ee14f3ad-53ca-4693-ab92-3aadcac65319)

Thanks

Csaba

**Comment by syssi** [comment]
Could you take a look at the Home Assistant log? Do you see any errors here?

**Comment by csababalogh** [comment]
> Could you take a look at the Home Assistant log? Do you see any errors here?

No, absolutely nothing under Home Assistant Core, no Xiaomi realted things.
Can be a problem that the Xiaomi Smart Plugs and the HASS VM are in different VLANs?

**Comment by syssi** [comment]
Yes. Some Xiaomi devices cannot be discovered across subnets: https://python-miio.readthedocs.io/en/latest/troubleshooting.html#discover-devices-across-subnets
